packages/ckeditor5-markdown-gfm/tests/gfmdataprocessor/code.js

```diff
@@ -273,7 +273,7 @@ describe( 'GFMDataProcessor', () => {
 				'<p><code>code `` code ```</code></p>',
 
 				// When converting back ````` will be normalized to ``.
-				'`code `` code ``` `'
+				'` code `` code ``` `'
```

Comment:
I tested few html-to-markdown converts available online and they are not consistent. I got the following results when converting `<p><code>code `` code ```</code></p>` to markdown:

```markdown
` code `` code ``` ` (new result)
`code `` code ``` ` (old result)
`code `` code ````
```

The first two results can be converted back to identical HTML, while the last one is incorrect:

```html
<p><code>code `` code ``` </code></p> (new result)
<p><code>code `` code ``` </code></p> (old result)
<p>`code `` code ````</p>
```

Because the "before" and "after" results are identical, I think this change is okay.

Reply:
> I tested few html-to-markdown converts available online and they are not consistent.

The GFMD stands for "GitHub Flavored Markdown" so we should be aligned with it.